### PR TITLE
Added links for the projects section issue #267

### PR DESCRIPTION
**Pull request by popcandy1x**
Added links for the projects section and also added a red hover for the links

### CODE REVIEW

**Comment by Harshil-Jani** [comment]
Hey @popcandy1x The links and the styling effect are working great thanks ! Is it possible for you to bring the text in the center of the Box ?
image

Also, The links open on the same tab, Instead convert them to open in the new Tab.

File: index.html [hunk 1]
```diff
@@ -207,21 +207,21 @@ <h2 style="text-align:center;">Inclusion</h2>
...
       <div class="column">
-        <h3>8051</h3>
-        <h3>ARM</h3>
-        <h3>Adafruit</h3>
-        <h3>Arduino</h3>
-        <h3>Atmel</h3>
-        <h3>Beagle</h3>
+        <h3><a href="https://github.com/prathimacode-hub/IoT-Spot/tree/main/8051">8051</a></h3>
+        <h3><a href="https://github.com/prathimacode-hub/IoT-Spot/tree/main/ARM">ARM</a></h3>
+        <h3><a href="https://github.com/prathimacode-hub/IoT-Spot/tree/main/Adafruit">Adafruit</a></h3>
```
Comment (Harshil-Jani) [inline index.html:212]: Open Links in New Tab

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (popcandy1x) [reply index.html:212]: Sure👍 I would do it

**Review by Harshil-Jani — APPROVED** [review]
Looks good to me 💯

**Pull request merged**